tpv_overwrite.yml

```diff
@@ -0,0 +1,840 @@
+## This file is a local EU overwrite of the global community TPV shared database of rules for the Total Perspective Vortex
+tools:
+  # local tools
+  upload1:
+    cores: 1
+    mem: 3
+    gpus: 0
+    env:
+      TEMP: /data/1/galaxy_db/tmp
+    scheduling:
+      require:
+      - condor_upload
+      - upload
+  __DATA_FETCH__:
+    cores: 1
+    mem: 3
+    gpus: 0
+    env:
+      TEMP: /data/1/galaxy_db/tmp
+    scheduling:
+      require:
+      - condor_upload
+      - upload
+  __SET_METADATA__:
+    cores: 1
+    mem: 1
+    gpus: 0
+    scheduling:
+      reject:
+      - pulsar
+    ## TODO this is to investigated
+    params:
+      requirements: GalaxyTraining == false
+      rank: GalaxyGroup == "metadata"
+    context:
+      params:
+        requirements: GalaxyTraining == false
+        rank: GalaxyGroup == "metadata"
+
+  interactive_tool_jupyter_notebook:
+    mem: 4
+    env:
+      HOME: /home/jovyan
+    scheduling:
+      require:
+      - condor_docker_ie_interactive
+      - docker
+    context:
+      force_destination_id: true
+  interactive_tool_pangeo_notebook:
+    mem: 4
+    env:
+      HOME: /home/jovyan
+    scheduling:
+      require:
+      - condor_docker_ie_interactive
+    context:
+      force_destination_id: true
+  interactive_tool_climate_notebook:
+    mem: 4
+    env:
+      HOME: /home/jovyan
+    scheduling:
+      require:
+      - condor_docker_ie_interactive
+    context:
+      force_destination_id: true
```

Comment:
`force_destination_id` appears to be a sorting hat specific param. I think we discussed adding support for directly specifying/templating destination name in TPV. Something like:
```
params:
   destination: my-dyn-dest-with-{cores}-{mem}-runner
```
So this function in sorting hat can be simulated or customised: https://github.com/usegalaxy-eu/sorting-hat/blob/0b0758a1b8b72bc0ea5ae198ad2949f4aa16b586/sorting_hat.py#L186
I'll add a test for this.